config/config.go

```
@@ -60,11 +61,19 @@
 
 // newConfig instantiate a new Config object
 func newConfig(format string) *Config {
+	codecRegistry := viper.NewCodecRegistry()
+	codec := hcl.Codec{}
+	codecRegistry.RegisterCodec("hcl", codec)
```

golangci-lint (1.64.8)

Error return value of `codecRegistry.RegisterCodec` is not checked

(errcheck)